msr-flows/Notes/type_1_flow.ipynb

```diff
@@ -0,0 +1,152 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Cycle Time Decay Approximation\n",
+    "\n",
+    "#### Suitable for BOC/SS/EOC\n",
+    "\n",
+    "This approximation treats the type 1 flow as a decay (which is how it is viewed mathematically). This means that the \"decay\" constant can be calculated from the \"half-life\" of a particular target.\n",
+    "\n",
+    "In this case, the half-lives are determined from the cycle times from Robertson et al. report on the MSBR that Andrei cites in the MSBR example.\n",
+    "\n",
+    "![title](cycle_time.PNG)\n",
+    "\n",
+    "| Element | Cycle Time (s) | \"Half-life\" | \"Decay\" Constant |\n",
+    "| :- | :-: | :-: | :-: | \n",
+    "| Kr | 20 | 10 | 6.93147E-2 |\n",
+    "| Xe | 20 | 10 | 6.93147E-2 |\n",
+    "| Se | 20 | 10 | 6.93147E-2 |\n",
+    "| Nb | 20 | 10 | 6.93147E-2 |\n",
+    "| Mo | 20 | 10 | 6.93147E-2 |\n",
+    "| Tc | 20 | 10 | 6.93147E-2 |\n",
+    "| Ru | 20 | 10 | 6.93147E-2 |\n",
+    "| Rh | 20 | 10 | 6.93147E-2 |\n",
+    "| Pd | 20 | 10 | 6.93147E-2 |\n",
+    "| Ag | 20 | 10 | 6.93147E-2 |\n",
+    "| Sb | 20 | 10 | 6.93147E-2 |\n",
+    "| Te | 20 | 10 | 6.93147E-2 |\n",
+    "| Cd | 17280000 | 8640000 | 8.02254E-8 |\n",
+    "| In | 17280000 | 8640000 | 8.02254E-8 |\n",
+    "| Sn | 17280000 | 8640000 | 8.02254E-8 |\n",
+    "| Br | 5184000 | 2592000 | 2.67418E-7 |\n",
+    "| I | 5184000 | 2592000 | 2.67418E-7 |\n",
+    "| Pa | 259200 | 129600 | 5.34836E-6 |\n",
+    "| Y | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| La | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| Ce | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| Pr | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| Nd | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| Pm | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| Sm | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| Gd | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| Eu | 4320000 | 2160000 | 3.20901E-7 |\n",
+    "| Rb | 296784000 | 148392000 | 4.67105E-9 |\n",
+    "| Sr | 296784000 | 148392000 | 4.67105E-9 |\n",
+    "| Cs | 296784000 | 148392000 | 4.67105E-9 |\n",
+    "| Ba | 296784000 | 148392000 | 4.67105E-9 |\n",
+    "\n",
+    "\n",
+    "These \"decay\" constants are then plugged in as the reprocessing constants\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Isotope-Based Linear Growth Approximation\n",
+    "\n",
+    "#### Only suitable for SS\n",
+    "\n",
+    "This approximation selects the \"most important\" isotopes from each element removal and sets the removal rate based on the Bateman equation for that isotope.\n",
+    "\n",
+    "Since some elements may not have isotopes which are necessarily important enough to track that closely (for neutronics purposes), we will set any element without important isotopes to use the base Linear Growth Approximation.\n",
+    "\n",
+    "https://www.osti.gov/etdeweb/servlets/purl/20332542 \n",
```

Comment:
It's not clear to me what this link is for. Perhaps you could user hypertext to clarify your intent?